### ISSUE TITLE
2.1.11 breaks flip

### ISSUE THREAD

**Issue by geddski** [pixijs/pixi-spine#366]
when upgrading from `2.1.10` to `2.1.11` I noticed my spine objects get distorted when they're flipped horizontally (`skeleton.scaleX = -1;`)

2.1.10
![image](https://user-images.githubusercontent.com/515722/102410964-83060000-3fae-11eb-8b60-974b32be1950.png)

2.1.11
![image](https://user-images.githubusercontent.com/515722/102410987-8b5e3b00-3fae-11eb-90a4-a6ca5796e164.png)

Reverting pixi-spine to `2.1.10` fixes the problem for me. Not sure what changed in pixi-spine to cause this.

**Comment by ivanpopelyshev** [comment]
Oh, that's interesting. Any way to reproduce it and make small example please? I honestly dont use spine at all, i just pull changes from spine-ts and ask people if it works for them :)

**Comment by geddski** [comment]
Yeah I'll make an example to test against. Did any of pixi-spine's dependencies change in `2.1.11`?

**Comment by ivanpopelyshev** [comment]
all deps i have are only for build. pixi-spine and spine-ts on its own dont have any deps

**Comment by ivanpopelyshev** [comment]
@geddski please?

**Comment by geddski** [comment]
@ivanpopelyshev oh thanks for the reminder! I'll whip it up now.

**Comment by geddski** [comment]
@ivanpopelyshev working on it, but having a hard time creating a demo. Pixi loader seems incompatible with services like codesandbox. So I'm switching to a local repo but having trouble now with pixi-spine's requirement of a global `PIXI` object.

**Comment by geddski** [comment]
@ivanpopelyshev if you have a demo I can modify I'd be happy to. Or if you'd rather, here is a spine object where you can see the distortion when flipped. (worked fined in pixi-spine 2.1.10 but not in 2.1.11)

https://drive.google.com/file/d/1CtxTDH3FCNtFsI2i_b61mDsnbEpmhSAq/view?usp=sharing, https://drive.google.com/file/d/1WSgpzp4UFymvRSv1qOG6hqWBHdtYl8GH/view?usp=sharing, https://drive.google.com/file/d/1cgRdyX-zSVxyjvnHUP5B_sBAn60knhQO/view?usp=sharing

**Comment by ivanpopelyshev** [comment]
Thanks! Is it buggy in starting state or in particular animation?

**Comment by geddski** [comment]
it's when it's flipped horizontally:

```js
hoodie.skeleton.scaleX = -1;
```

I've noticed it in the idle animation:

```js
hoodie.state.setAnimation(0, 'idle', true);
```

**Comment by geddski** [comment]
@ivanpopelyshev did you have a chance to take a look?

**Comment by ivanpopelyshev** [comment]
No, I was on vacation :) gonna look at it today

**Comment by ivanpopelyshev** [comment]
OK, after all my adventures, im here. 

OK, reproduced the bug, looking at it

**Comment by ivanpopelyshev** [comment]
Here it is: https://github.com/pixijs/pixi-spine/commit/70e3529f4ca96254ad54c0d0eba4f259846fb9b7#diff-1a966784a6695720504c42a12edab4ce61e80f435c1eee0caf0ab8348201baf2R150

**Comment by ivanpopelyshev** [comment]
I dont know why it was closed :)
